src/Standards/Squiz/Tests/Commenting/ClosingDeclarationCommentUnitTest.4.inc

```diff
@@ -0,0 +1,8 @@
+<?php
```

Comment:
While this is a good test to have anyway, this test doesn't actually highlight the problem with the missing `$next !== false` condition.

We looked at this during our call and the following additional test should be added to safeguard the fix:
```php
//end closingBraceAtEndOfFileMissingComment()

<?php

function closingBraceAtEndOfFileMissingComment() {
}
```

This test will show that without that extra condition the _wrong_ error is being thrown (though an error is still being thrown).


_Note: as the test framework as-is doesn't safeguard that the correct error code is being thrown for each line, it's not a hard safeguard, but having the test in place will allow us to safeguard this properly if/when the test framework gets updates to check per error code/per line._